README.md

```diff
@@ -0,0 +1,20 @@
+# jordanthepowerful-sdk
+An SDK for the Lord of the Rings Trilogy API 
+If you don't hire me, I will go to another company to bring chaos and mayhem to them and you prolly don't want that on your concious. Thing of the OTHER COMPANIES and how they will have to deal with me. Kinda sad right? **can you hear arms of an angel playing in the background?
+
+# Purpose 
+This is to be a wrapper around an existing API that will hopefully encapsulate and decouple logic in way that makes it easier for the future engineers 
+
+# Scope 
+I am only handling the movie and the qoute parts of the API if you require additional functionality, well , that sucks for you. 
+
+# Setup
+Just like most packages its just "npm install jordanthepowerful-sdk"
+
+# Testing 
+I was not getting paid enough when building this to do test driven development, please use at your own risk. 
```

Comment:
this is supposed to be a showcase of your skills no? test driven should be the default way.